src/main/java/de/tum/in/www1/artemis/domain/exam/StudentExam.java

```diff
@@ -84,6 +85,12 @@ public class StudentExam extends AbstractAuditingEntity {
     @JoinTable(name = "student_exam_quiz_question", joinColumns = @JoinColumn(name = "student_exam_id", referencedColumnName = "id"), inverseJoinColumns = @JoinColumn(name = "quiz_question_id", referencedColumnName = "id"))
     private List<QuizQuestion> quizQuestions = new ArrayList<>();
 
+    @ManyToMany
```

Comment:
Why does this have to be ManyToMany? Doesn't one student_exam have multiple participations, but a participation only has one student_exam? What am I missing here?

Reply:
Yes, you are right. However, we made this relationship ManyToMany on purpose, since we want to have a separate table to store this relationship. If we made this OneToMany, this would add a column student_exam to participation table, which would be empty/null in most of the cases.